src/rdo.rs

```diff
@@ -147,10 +147,23 @@ fn compute_rd_cost(
     unimplemented!();
   };
 
+  let PlaneConfig { xdec, ydec, .. } = fs.input.planes[1].cfg;
+
+  let mask = !(MI_SIZE - 1);
+  let mut w_uv = (w_y >> xdec) & mask;
+  let mut h_uv = (h_y >> ydec) & mask;
+
+  let bo_chroma = BlockOffset { x: bo.x & !xdec, y: bo.y & !ydec };
```

Comment:
I did not get why bo_chroma need truncation if chroma is not sampled in either x or y.
If chroma is 444, xdec == ydec == 0, then bo_chroma.x|y can only locate the size 8x8 blocks when each 4x4 luma has its corresponding 4x4 chromas blocks?